app/javascript/onboarding/__tests__/Onboarding.test.jsx

```diff
@@ -76,13 +76,16 @@ describe('<Onboarding />', () => {
   ]);
   const dataUser = JSON.stringify({
```

Comment:
Looking at the existing tests, none of them mutate this data, but it may be wise to always return new user data.

```suggestion
  const getUserData = () => JSON.stringify({
```

and update the tests, e.g.

```javascript
beforeEach(() => {
     onboardingSlides = initializeSlides(0, getUserData());
});
```

Reply:
Ahhhhh, yeah. This should prevent us from running into this in the future. I think @maestromac figured out that there is no proper tear-down for this so that's why `dataUser` looked as expected for later tests in this file, but not for the initial test.

I'm going to update with your change!

Reply:
That's a great suggestion @nickytonline !